Patch-Source: https://github.com/hercules-team/ruby-augeas/commit/89652bd605ee7b860536a9157cf99854d42c7a6d.patch
--
From 89652bd605ee7b860536a9157cf99854d42c7a6d Mon Sep 17 00:00:00 2001
From: Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
Date: Sun, 28 Jul 2024 09:33:09 +0200
Subject: [PATCH] Remove unused sibling argument from augeas_rm (#17)

This makes the function match the method definition. There was already a
warning about this, but Fedora 40 applies more hardening and it's a
fatal error.

Fixes: e6b57c58d24b ("Initial ruby bindings, written by Bryan Kearney")
---
 ext/augeas/_augeas.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/augeas/_augeas.c b/ext/augeas/_augeas.c
index f9b49d1..7ef0d7d 100644
--- a/ext/augeas/_augeas.c
+++ b/ext/augeas/_augeas.c
@@ -184,7 +184,7 @@ VALUE augeas_mv(VALUE s, VALUE src, VALUE dst) {
  *
  * Remove path and all its children. Returns the number of entries removed
  */
-VALUE augeas_rm(VALUE s, VALUE path, VALUE sibling) {
+VALUE augeas_rm(VALUE s, VALUE path) {
     augeas *aug = aug_handle(s);
     const char *cpath = StringValueCStr(path) ;
 
